local/env.go

```diff
@@ -1,6 +1,42 @@
 package local
 
+import (
+	"errors"
+	"os"
+	"runtime"
+)
+
 const (
 	// EnvSSHPassphrase is the key used to fetch PEM key passphrases
 	EnvSSHPassphrase = "PEM_PASSPHRASE"
 )
+
+// GetHomePath returns the homepath based on the operating system used
+func GetHomePath() (string, error) {
+	if runtime.GOOS == "windows" {
+		// Performs check for HOME env variable first
+		if home := os.Getenv("HOME"); home != "" {
+			return home, nil
+		}
+
+		// Performs check for USERPROFILE as backup
+		if home := os.Getenv("USERPROFILE"); home != "" {
+			return home, nil
+		}
+
+		// Builds HOME from HOMEDRIVE and HOMEPATH as default
+		drive := os.Getenv("HOMEDRIVE")
+		path := os.Getenv("HOMEPATH")
```

Comment:
nitpicking, but for standalone, single-variable assignments I prefer the use of `var` for clarity (ie make it clear that a new variable is assigned created here)

```suggestion
		var path = os.Getenv("HOMEPATH")
```

not a big deal though!